components/Footer.tsx

```diff
@@ -1,6 +1,7 @@
 import Image from "next/image";
 import Link from "next/link";
-import ICAF_logo from "../public/icaf-logo-transparent.svg";
+import ICAF_logo_desktop from "../public/icaf-logo-transparent.svg";
```

Comment:
I've never seen this component naming convention used with React. Unless I'm missing something, please use pascal case (ex. `ICAFLogoDesktop`)